design/one-pager-managed-resource-api-design.md

```diff
@@ -379,6 +379,149 @@ type SubnetworkParameters struct {
 }
 ```
 
+### Exceptions and Edge Cases
+
+Because managed resources could represent API types for any provider, there is
+potential to encounter scenarios that fall outside the rules expressed in this
+design document. Some common exceptions and how to address them are expressed
+below.
+
+#### Embedded Structs with Mixed Fields
+
+Some provider APIs include an embedded struct that may contain some fields that
```

Comment:
I think the section `High Fidelity` is the most relevant one, so, maybe we could move this up there as an example case.